### PR TITLE
RHDEVDOCS-4175 Document that all S2I tasks are failing and the solution

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.9`, `enterprise-4.10`, `enterprise-4.11`, `enterprise-4.12`
- **JIRA issues**: [RHDEVDOCS-4175 Document that all S2I build tasks are failing and solutions](https://issues.redhat.com/browse/RHDEVDOCS-4175)
- **Preview pages**: See the `Fixed issues` subsection in the 1.7.2 and 1.8 sections:
  - **1.8**: http://file.pnq.redhat.com/sosarkar/4175-S2I-tasks-failing/cicd/pipelines/op-release-notes.html#fixed-issues-1-8_op-release-notes
  - **1.7.2**: http://file.pnq.redhat.com/sosarkar/4175-S2I-tasks-failing/cicd/pipelines/op-release-notes.html#fixed-issues-1-7-2_op-release-notes
- **SME review**: @piyush-garg          
- **QE review**: Not necessary as this is a very minor change that doesn't affect any procedure.
- **Peer-review**: TBD

### CODE REVIEW

File: modules/op-release-notes-1-7.adoc [hunk 1]
```diff
@@ -410,4 +410,14 @@ time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
 ----
 +
 With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks run successfully.
```
Comment (piyush-garg) [inline modules/op-release-notes-1-7.adoc:412]: as a result, the S2I buildah and buildah tasks can run successfully

Comment (piyush-garg) [reply modules/op-release-notes-1-7.adoc:412]: or may be you can remove this line, as it is mentioned again in next line

Comment (sounix000) [reply modules/op-release-notes-1-7.adoc]: Resolved.

File: modules/op-release-notes-1-7.adoc [hunk 2]
```diff
@@ -409,5 +409,15 @@ time="2022-03-04T09:47:57Z" level=error msg="error writing \"0 0 4294967295\\n\"
 time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
 ----
 +
-With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks run successfully.
+With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks can run successfully.
```
Comment (piyush-garg) [inline modules/op-release-notes-1-7.adoc:412]: as a result, the `Buildah and` S2I `tasks`

File: modules/op-release-notes-1-8.adoc [hunk 3]
```diff
@@ -494,13 +494,23 @@ This update fixes the issue. It passes the namespace-based configuration to the
...
 +
+[source,terminal]
 ----
 Error: error writing "0 0 4294967295\n" to /proc/22/uid_map: write /proc/22/uid_map: operation not permitted
 time="2022-03-04T09:47:57Z" level=error msg="error writing \"0 0 4294967295\\n\" to /proc/22/uid_map: write /proc/22/uid_map: operation not permitted"
 time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
 ----
 +
-With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah cluster tasks. As a result, the S2I build tasks run successfully.
+With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks can run successfully.
```
Comment (piyush-garg) [inline modules/op-release-notes-1-8.adoc:504]: as a result, the `Buildah and` S2I `tasks`

File: modules/op-release-notes-1-7.adoc [hunk 4]
```diff
@@ -409,7 +409,17 @@ time="2022-03-04T09:47:57Z" level=error msg="error writing \"0 0 4294967295\\n\"
 time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
 ----
 +
-With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks run successfully.
+With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for `Buildah` and `S2I` clustertasks. As a result, the `Buildah` and `S2I` build tasks can run successfully.
++
+To successfully run the `Buildah` clustertask and S2I build tasks for applications written in various languages and frameworks, add the following snippet for appropriate `steps` object such as `build` and `push`:
```
Comment (gabriel-rh) [inline modules/op-release-notes-1-7.adoc:414]: Maybe tick the S2I text here (and below in the other file) to keep it consistent with the rest of the content:

`S2I`

Comment (gabriel-rh) [inline modules/op-release-notes-1-7.adoc:414]: for appropriate `steps` object:

Maybe either:

"for the appropriate `steps` object" or "for appropriate `steps` objects"  ?

**Review by gabriel-rh — COMMENTED** [review]
looks good, only 2 very minor suggestions

**Review by gabriel-rh — APPROVED** [review]
/lgtm

Looks good!

File: modules/op-release-notes-1-7.adoc [hunk 5]
```diff
@@ -409,7 +409,17 @@ time="2022-03-04T09:47:57Z" level=error msg="error writing \"0 0 4294967295\\n\"
 time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
 ----
 +
-With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks run successfully.
+With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for `Buildah` and `S2I` clustertasks. As a result, the `Buildah` and `S2I` build tasks can run successfully.
```
Comment (rolfedh) [inline modules/op-release-notes-1-7.adoc:412]: Please replace instances of `clustertask` that are not objects or commands with `cluster task`. Searching the openshift-docs repo confirms this pattern.

Comment (rolfedh) [reply modules/op-release-notes-1-7.adoc:412]: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references

**Review by rolfedh — COMMENTED** [review]
Please make this minor update before merge. Otherwise, looks good to me.

File: modules/op-release-notes-1-7.adoc (hunk 5 above)
Comment (rolfedh) [reply modules/op-release-notes-1-7.adoc:412]: @gabriel-rh FYI ^^^

**Comment by sounix000** [comment]
@rolfedh: The manual cherry-picks:
- **4.9**: https://github.com/openshift/openshift-docs/pull/51585
- **4.10**: https://github.com/openshift/openshift-docs/pull/51586

**Pull request merged**